tests/contracts/EventDispatcher/Event/SuggestionEventTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Contracts\Search\Event;
+
+use Ibexa\Contracts\Search\Event\SuggestionEvent;
+use Ibexa\Contracts\Search\Model\Suggestion\SuggestionCollection;
+use Ibexa\Search\Model\SuggestionQuery;
+use PHPUnit\Framework\TestCase;
+
+final class SuggestionEventTest extends TestCase
+{
+    public function testConstruct(): void
+    {
+        $suggestionQuery = new SuggestionEvent(new SuggestionQuery('test', 3));
+
+        self::assertInstanceOf(SuggestionCollection::class, $suggestionQuery->getSuggestionCollection());
+        self::assertInstanceOf(SuggestionQuery::class, $suggestionQuery->getQuery());
```

Comment:
Those assertions can be skipped, the argument implicitly declares the return type.
Instead, you could check:
```
        $suggestionQuery = new SuggestionQuery('test', 3);
        $suggestionEvent = new SuggestionEvent($suggestionQuery);
        $this->assertSame($suggestionQuery, $suggestionEvent->getQuery());
        $this->assertEquals(new SuggestionCollection(), $suggestionEvent->getSuggestionCollection());
```